src/utils.ts

```diff
@@ -13,8 +13,11 @@ export class Utils {
         if (matches && matches.length > 1) {
             switch (resultType) {
                 case "name":
+                    // Removes "operations/"" from name in case of API named "operations"
+                    if (matches[1].includes('operations/')) {
+                        return matches[1].replace('operations/', '');
+                    }
                     return matches[1];
```

Comment:
Thanks for the fix! 😄 

I don't think we need the `if` here.
Also, maybe, instead of replacing `operations/` we could replace the resource directly. this way we ensure that the the name is returned correctly for other possible corner cases.
```suggestion
                   case "name":
                      // Removes "<resource>/"" from name in case it matches a reserved word"
                    return matches[1].replace(`${resource}/`, '');
```

Reply:
I guess this might break if the name ends with "resource" string
e.g. /apis/my-api/operations/our-company-operations/foo
the desired result would not be `our-company-foo`

Reply:
Thanks for the suggestions - I can re-test later and come back with a better solution. Or if you have an alternate fix that you want to apply, please let me know and we can cancel this PR.

I think that is right, I did not consider you could have a "/" in the actual operation name as in: "/apis/my-api/operations/our-company-operations/foo" - just to double check, the desired result in this case would be:

"our-company-operations/foo"

Reply:
@JMach1 Just seen your comment on the Issue - closing this.